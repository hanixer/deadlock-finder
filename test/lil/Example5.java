package showcase;

/** 
 * Call statement
 */
public class Example2 {
    public static void func1() {
        int a = 1;
        int b = 2;
        func2(a, c);
    }

    public static void func2(int x, int y) {
        x = x - y;
    }
}
