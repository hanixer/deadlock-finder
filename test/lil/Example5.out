func showcase.Example2.func1(c: int): void
entry:
a = 1
b = 2
showcase.Example2.func2
jump end

end:
return

func showcase.Example2.func2(x: int, y: int): void
entry:
x = x-y
jump end

end:
return